README.md

```diff
@@ -1,9 +1,10 @@
 [![Netlify Status](https://api.netlify.com/api/v1/badges/7cdc05a8-9023-4af3-8f04-4b8b63ec51f3/deploy-status)](https://app.netlify.com/sites/juggernaut/deploys)
-[![All Contributors](https://img.shields.io/badge/all_contributors-3-orange.svg?style=flat-square)](#contributors)
+[![All Contributors](https://img.shields.io/badge/all_contributors-5-orange.svg?style=flat-square)](#contributors)
+![Lint and test](https://github.com/SoftboxLab/juggernaut/workflows/Lint%20and%20test/badge.svg)
```

Comment:
Esse link me parece estar incorreto.

Reply:
Ajustado. Tirei o link do status de workflow de lint and test.